gcalbot/gcalbot/reminderscheduler/send.go

```diff
@@ -0,0 +1,70 @@
+package reminderscheduler
+
+import (
+	"time"
+
+	"github.com/keybase/managed-bots/gcalbot/gcalbot"
+)
+
+// reference: https://stackoverflow.com/a/39295990
+type ReminderTicker struct {
+	*time.Timer
+}
+
+func NewReminderTicker() *ReminderTicker {
+	return &ReminderTicker{time.NewTimer(getNextTickDuration())}
+}
+
+func (rt *ReminderTicker) Update() {
+	rt.Reset(getNextTickDuration())
+}
+
+func getNextTickDuration() time.Duration {
```

Comment:
why not `const nextTickDuration = time.Minute`?

Reply:
My reason for doing this (based off of the SO post commented above) is so that the reminderloop executes exactly (more or less) on each minute. Otherwise it might start straying by a bit since the delay is one minute + the execution time for the reminders, and although that isn't necessarily a bad thing I want to make sure no minutes are skipped.